chcemvediet/apps/accounts/admin.py

```diff
@@ -1,15 +1,35 @@
 # vim: expandtab
 # -*- coding: utf-8 -*-
+from django import forms
 from django.contrib import admin
+from django.core.exceptions import ValidationError
 
 from poleno.utils.misc import decorate
 from poleno.utils.admin import simple_list_filter_factory, admin_obj_format
 
 from .models import Profile
 
 
+class ProfileAdminForm(forms.ModelForm):
+
+    class Meta:
+        model = Profile
+        fields = u'__all__'
```

Comment:
Totu meta class v model forme realne v adminovi netreba, admin tam automaticky doplni defaultne hodnoty z `ProfileAdmin`. Bolo by to treba, iba ak by sme chceli defaultne hodnoty overridnut.